daemons/controld/controld_attrd.c

```diff
@@ -117,7 +117,7 @@ update_attrd_remote_node_removed(const char *host, const char *user_name)
     if (rc == pcmk_rc_ok) {
         crm_trace("Asking attribute manager to purge Pacemaker Remote node %s",
                   host);
-        rc = pcmk__attrd_api_purge(attrd_api, host, true);
+        rc = pcmk__attrd_api_purge(attrd_api, host);
```

Comment:
It's only always true for the moment.  https://github.com/ClusterLabs/pacemaker/pull/3801/files#diff-0b476b06066c8ec40a691cffa57e026dfc6552a9ae606404fcb03e2307939f1cR128 can pass both true and false.

Reply:
re-pushed without this commit